code/modules/mob/living/simple_animal/hostile/mining_mobs/mining_mobs.dm

```diff
@@ -37,13 +37,13 @@
 		return
 	icon_state = icon_living
 
-/mob/living/simple_animal/hostile/asteroid/bullet_act(obj/item/projectile/P)//Reduces damage from most projectiles to curb off-screen kills
-	if(!stat)
-		Aggro()
-	if(P.damage < 30 && P.damage_type != BRUTE)
-		P.damage = (P.damage / 3)
-		visible_message("<span class='danger'>[P] has a reduced effect on [src]!</span>")
-	..()
+//mob/living/simple_animal/hostile/asteroid/bullet_act(obj/item/projectile/P)//Reduces damage from most projectiles to curb off-screen kills
+//	if(!stat)
+//		Aggro()
```

Comment:
Removing Aggro() will make the fauna passive when shot. MAXIMUM CHEESE with anything ranged.

Reply:
I have yet again made grave mistakes.